src/tests/admin/conferenceroomoutoforderplanning/OutOfOrderIsNotCreatedIfStartTimeAndEndTimeAreInThePast.java

```diff
@@ -0,0 +1,62 @@
+package tests.admin.conferenceroomoutoforderplanning;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.json.JSONException;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomOutOfOrderPlanningPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC22: Verify an Out Of Order cannot be created when the start time and end time are in the past
+ * TC31: Verify when the start time and end time are in the past the following message is displayed 
+ * "Cannot establish out of order as a past event"
+ * @author Yesica Acha
+ *
+ */
+public class OutOfOrderIsNotCreatedIfStartTimeAndEndTimeAreInThePast {
+	RoomOutOfOrderPlanningPage outOfOrderPage;
+	ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	List<Map<String, String>> testData = excelReader.getMapValues("OutOfOrderPlanning");
```

Comment:
Same situation here, All of your variables are used as local variables, but declared as global. Please declare them inside test method